### DISCUSSION TITLE
Reinstatement Open For Almost 60 Days

### DISCUSSION THREAD

**Discussion by GoodbyeKittyy** [community/community#113682] [Discussions]
### Select Topic Area

Question

### Body

Hi everyone,

I made a reinstatement request previously and it is almost 60 days.. I can't transfer my repos too as when I tried to, there was a "Repository transfers not available for this account" message.

I have about 30 repos others need to assess.. If any support staff come across this, please refer to request #2618814

**Comment by ettaboyle** [comment]
Hi @GoodbyeKittyy,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post is being locked and closed because it contained content that violates our Community Code of Conduct, **including posting the same comment in multiple places**. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post. ex: https://github.com/orgs/community/discussions/113684

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
